### PR TITLE
Feature/cdcinitializer 161

### PR DESCRIPTION
**Pull request by bmmsantos**

### CODE REVIEW

File: cdc-accelerator/setup/installer.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        Object.entries(acceleratorPackageJson[dependenciesProperty]).forEach((entry) => {
+            if (!this.#containsForbiddenCharacters(entry[0])) {
+                Object.assign(newProjectPackageJson[dependenciesProperty], { [entry[0]]: entry[1] })
+            }
+        })
+    }
+
+    #containsForbiddenCharacters(dependency) {
+        const forbidden = dependency.match(/[*?:;,&|+]/)
+        return forbidden ? true : false
```
Comment (danielsc83) [inline cdc-accelerator/setup/installer.js:35]: I would simplify this to: return dependency.match(/[*?:;,&|+]/) ? true : false

File: cdc-accelerator/setup/uninstaller.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            ) {
+                spawnSync(`npm remove ${entry[0]}`, { shell: false, stdio: 'inherit' })
+            }
+        })
+        this.#uninstallDependency('light-server', newProjectPackageJson)
+    }
+
+    #containsForbiddenCharacters(dependency) {
+        const forbidden = dependency.match(/[*?:;,&|+]/)
+        return forbidden ? true : false
```
Comment (danielsc83) [inline cdc-accelerator/setup/uninstaller.js:28]: same as above

Comment (bmmsantos) [reply cdc-accelerator/setup/uninstaller.js:28]: Agree

File: cdc-accelerator/setup/installer.js (hunk 1 above)
Comment (bmmsantos) [reply cdc-accelerator/setup/installer.js:35]: agree

**Review by danielsc83 — APPROVED** [review]

**Pull request merged**